blivet/formats/disklabel.py

```diff
@@ -324,6 +324,15 @@ def logicalPartitions(self):
         return logicals
 
     @property
+    def primaryPartitions(self):
+        try:
+            primaries = self.partedDisk.getPrimaryPartitions()
+        except Exception: # pylint: disable=broad-except
```

Comment:
Do we really need this broad exception? It seems to me like it will be hiding real bugs in the code.